samigo/samigo-api/src/java/org/sakaiproject/samigo/util/SamigoConstants.java

```diff
@@ -40,14 +40,60 @@
     /*
      * Events
      */
+    // Submission events
```

Comment:
While your here should probably make the class final and add a private constructor so this class cannot be instantiated! This is usually the best accepted pattern for classes containing constants.